modbus/README.md

```diff
@@ -0,0 +1,30 @@
+## Modbus Monitoring Template
+
+This InfluxDB Template can be used to gather data from a Modbus slave
+
+![Website Monitoring Dashboard Screenshot](img/Dashboard.png)
+
+### Included Resources
+
+- 1 label: `island_pulse`
+- 1 bucket: 'node8'
+- 9 Alerts: Threshold checks for all areas in `area_names`
+- 2 Variables: `area_names`, `energy_types`
+- 1 Dashboard: `Island Pulse`  (see screenshot above)
+
+## Setup Instructions
+
+General instructions on using InfluxDB Templates can be found in the [use a template](../docs/use_a_template.md) document.
+
+### Telegraf
+  See `[[inputs.modbus]]` section in this template for [more information](./modbus.yml).
```

Comment:
Please provide more in-depth instructions for how to run Telegraf, including any modifications needed to the configuration file and anything that needs to be configured in Island Pulse to allow it to read data

Reply:
Let me know what you think of it now.

btw, I created a repository for a demo device that the community can use along with this template. it's referenced in the readme